src/style.css

```diff
@@ -1 +1,112 @@
-/* add styles here */
+html {
+  --main-color: var(--main-color);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir.ttc');
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url('fonts/Avenir-Book.ttf');
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url('fonts/Avenir-Heavy.ttf');
+}
+
+body {
+  box-sizing: border-box;
+  color: var(--main-color);
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.search-bar__input {
+  width: 100%;
+}
+
+.search-bar__input--big {
+  height: 70px;
+  padding-left: 62px;
+
+  box-sizing: border-box;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
```

Comment:
move on common styles of `.search-bar__input--small,  .search-bar__input--big` to `.search-bar__input` to reduce copy/past

Reply:
Moved on common styles of .search-bar__input--small,  .search-bar__input--big to .search-bar__input.